openedx/core/lib/api/tests/helpers.py

```diff
@@ -0,0 +1,78 @@
+"""
```

Comment:
Nit: Move this to openedx/core/lib/api/test_utils.py.

That would allow us to keep the `tests` directory specifically for actual tests.